styles/colors-silicon-day.less

```diff
@@ -206,13 +207,14 @@
 // ---------------------------------------
 @agate-item-border-color:                 fade(@agate-black, 40%);
 @agate-item-color:                        @agate-text-color;
-@agate-item-label-color:                  @agate-noble-gray;
-@agate-item-focus-bg-color:               fade(@agate-black, 15%);
+@agate-item-label-color:                  fade(@agate-foreground, 70%);
+@agate-item-label-focus-selected-color:   fade(@agate-foreground, 70%);
```

Comment:
🤔 I think... we don't want to set color with opacity here. The guide should be fixed. Let's not change these.